internal/services/compute/virtual_machine_scale_set.go

```diff
@@ -471,12 +457,7 @@ func ExpandVirtualMachineScaleSetScaleInPolicy(input []interface{}) *compute.Sca
 
 func FlattenVirtualMachineScaleSetScaleInPolicy(input *compute.ScaleInPolicy) []interface{} {
 	if input == nil {
-		return []interface{}{
```

Comment:
This causes diff when `scale_in` is not specified. saying updating these two values from their default values to nil, so returning an empty interface instead